src/PhpGenerator/Method.php

```diff
@@ -81,9 +88,10 @@ public static function from($from)
 	public function __toString()
 	{
 		$parameters = [];
+		static $builtinTypes = ['array', 'string', 'bool', 'float', 'int', 'callable', ''];
```

Comment:
I pushed support for callable / self / parent, so you can now simply add PHP 7 types fa4e80a0e25f593af0479d7bdd6862e461448906